.github/tests/images.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Some newly added images are missing here.

Reply:
Not sure which ones you mean? the kubectl ones don't have versions pinned.

Reply:
fsGroup was added since, so I just added it here.